rules/do-you-follow-best-practices-for-managing-microsoft-entra-id/rule.md

```diff
@@ -0,0 +1,79 @@
+---
+seoDescription: Learn best practices for managing Microsoft Entra ID to enhance security and efficiency in your organization.
+type: rule
+title: Do you follow best practices for managing Microsoft Entra ID?
+uri: do-you-follow-best-practices-for-managing-microsoft-entra-id
+authors:
+  - title: Rob Thomlinson
+    url: https://www.ssw.com.au/people/RobThomlinson
+created: 2025-01-03T10:58:08.000Z
+guid: 123e4567-e89b-12d3-a456-426614174000
+---
+
+Effective management of Microsoft Entra ID (formerly Azure Active Directory) is crucial for maintaining the security and efficiency of your organisation's IT infrastructure. Neglecting best practices can lead to unauthorised access, data breaches, and operational disruptions. <!--endintro-->
+
+**1. Enforce Strong Authentication**
+
+- **Implement Multi-Factor Authentication (MFA):** Require MFA for all users, especially administrators, to add an extra layer of security.
+
+- **Adopt Passwordless Authentication:** Utilise methods like Windows Hello for Business or FIDO2 security keys to enhance security and user experience.
+
+**2. Apply the Principle of Least Privilege**
+
+- **Use Role-Based Access Control (RBAC):** Assign users the minimum permissions necessary for their roles to reduce the risk of unauthorised access.
+
+- **Implement Just-In-Time Access:** Utilise Privileged Identity Management (PIM) to grant temporary access to resources only when needed.
+
+**3. Regularly Review and Audit Access**
+
+- **Conduct Access Reviews:** Periodically review user access to ensure that only authorised individuals have access to resources.
+
+- **Monitor Sign-In Activity:** Keep track of user sign-ins to detect unusual or suspicious activities promptly.
+
+**4. Secure Application Registrations**
+
+- **Use Certificates Over Secrets:** Always use certificate credentials for app authentication instead of client secrets, as certificates are more secure.
+
+- **Limit API Permissions:** Assign the least privileged permissions necessary for applications to function.
+
+**5. Enable Security Features**
+
+- **Activate Security Defaults:** Enable security defaults in Microsoft Entra ID to enforce a basic level of security across your organisation.
+
+- **Implement Conditional Access Policies:** Define policies that grant or block access based on conditions like user location, device state, or risk level.
+
+**6. Plan for Emergency Access**
+
+- **Create Break Glass Accounts:** Establish at least two emergency access accounts that are not protected by MFA to ensure access during critical situations.
+
+- **Monitor and Secure Emergency Accounts:** Regularly audit these accounts to ensure they are not misused and are only accessed during emergencies.
+
+### **7. Use Clear Access Group Naming Conventions**
+
+Clear and consistent naming conventions for access groups make management simpler and ensure clarity across the organisation.
+
+#### **Why Are Naming Conventions Important?**
```

Comment:
Hi @RobJT-wq 

These headings should NOT be capitalized (there is a rule saying only the main heading should)

- [ ] 1. Please fix this and others